src/core/balance-sheet.ts

```diff
@@ -21,11 +23,15 @@ export const updatePendingLightningTransactions = async () => {
   await runInParallel({
     iterator: usersWithPendingInvoices,
     logger,
-    processor: async ({ _id }, index) => {
-      logger.trace("updating pending invoices for user %s in worker %d", _id, index)
-      const user = await User.findOne({ _id })
-      const userWallet = await WalletFactory({ user, logger })
-      await userWallet.updatePendingInvoices()
+    processor: async ({ id }, index) => {
```

Comment:
you cant change to id, the param is _id because is an aggregate function (please check line 18)

Reply:
I've taken the commit changing this out of the history. I'd prefer to use the

```
const user = await User.findOne({ _id })
... user.id
```
method for now. I know this is less efficient but I am not sure wether or not passing `_id` as `walletId` will always work. My understanding wast that `_id` was always referring to a reference `Object` and `id` was always a string.